### ISSUE TITLE
Mouth doesn't export properly, keeps protruding

### ISSUE THREAD

**Issue by lurkingaround** [FlailingFog/KK-Blender-Shader-Pack#33]
I noticed this keeps happening when I export something from Koikatsu and don't know how to fix this, the mouth seems to keep protruding and doesn't look like how it was in Koikatsu, it has this fish shape to it now. Is there a way to fix this? This is the first time I'm exporting it and following the youtube tutorial but I noticed this keeps happening.

![mouth](https://user-images.githubusercontent.com/88926467/132323169-7c48e9ad-1534-4df5-9793-7395e543b9f7.png)

**Comment by FlailingFog** [comment]
That is probably a pmx exporter thing. If you can't get it to export correctly after loading the character maker, returning to the main menu and going into the maker a second time then you'll have to use the fbx method. 

For step 1f, I usually just scale it down until the height of the hand bones matches up with the height of the hands. Once you scale it and get out of edit mode, everything will snap into the correct place. After clicking the import button it should automatically select all of the bones and put you in armature edit mode with the cursor at the origin, so this should be easy to do.

**Comment by lurkingaround** [comment]
> That is probably a pmx exporter thing. If you can't get it to export correctly after loading the character maker, returning to the main menu and going into the maker a second time then you'll have to use the fbx method.
Can confirm it's a pmx exporter issue, I don't know why this happens, exporting using grey's mesh exporter doesn't have this bug.

> For step 1f, I usually just scale it down until the height of the hand bones matches up with the height of the hands. Once you scale it and get out of edit mode, everything will snap into the correct place. After clicking the import button it should automatically select all of the bones and put you in armature edit mode with the cursor at the origin, so this should be easy to do.
Thanks, I'm still new to blender and wasn't sure if what I was doing was correct or not.

**Comment by FlailingFog** [comment]
Yeah those are it. On the top, you need keep the transform pivot point as 3D cursor though. The entire armature needs to be scaled to that height with the 3D cursor at the origin
![image](https://user-images.githubusercontent.com/65811931/132921356-d2d53baa-e7d0-45b8-8d69-de238735323a.png)

**Comment by lurkingaround** [comment]
Thanks for assisting, it's challenging because I'm still new to blender and aren't used to the program yet, sorry for asking again, I'm assuming this is the correct for the armature. 

![blender](https://user-images.githubusercontent.com/88926467/132944322-501b0d80-d274-492d-8a22-46d2a59ac855.png)
